plugin/eleline.vim

```diff
@@ -204,6 +204,21 @@ function! ElelineVista() abort
   return !empty(get(b:, 'vista_nearest_method_or_function', '')) ? s:fn_icon.b:vista_nearest_method_or_function : ''
 endfunction
 
+function! ElelineNvimLsp() abort
+  if s:is_tmp_file()
+    return ''
+  endif
+  if luaeval('#vim.lsp.buf_get_clients() > 0')
+    let l:lsp_status = luaeval("require('lsp-status').status()")
```

Comment:
I'm guessing neovim-nightly is required?

Reply:
luaeval('#vim.lsp.buf_get_clients() > 0') will return 0 for vim and nvim-0.4